### ISSUE TITLE
Exploring Real-time Video Instance Segmentation with DVIS Model

### ISSUE THREAD

**Issue by mbk2103** [zhang-tao-whu/DVIS#21]
I am currently using the DVIS model for inference, and it appears to take a directory of video frames in image format as input. I would like to inquire whether it is possible to directly input a video for real-time video instance segmentation.

Is it feasible to configure the DVIS model to work with video input, allowing for real-time video instance segmentation, or is it limited to processing individual frames in image format?

Thank you for your guidance and support!

**Comment by zhang-tao-whu** [comment]
Yes, DVIS can accept video input, but it may require some code modifications. Additionally, when you mention video input, are you referring to inputs like mp4 files or real-time video stream?

**Comment by mbk2103** [comment]
Firstly,  thank you for your response.

Yes, I want to provide input in both ways, especially real-time video stream. My initial approach was to split the video into frames and provide the frames as input in an image format to the input folder, then combine the output frames to create a video. However, this is not an optimal option in terms of time efficiency.
